docs/DeveloperGuide.md

```diff
@@ -656,84 +678,153 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 * 1b. Input parameters is invalid.
 
-  * 1a1. TrackAScholar displays an error message showing parameter requirements. <br>
+  * 1b1. TrackAScholar displays an error message showing parameter requirements. <br>
     Use case resumes at step 1.
 
-* 1c. An applicant with the same name already exist.
+* 1c. An applicant with the same name already exists.
 
-  * 1a1 TrackAScholar displays an error message that applicant already exist. <br>
+  * 1c1 TrackAScholar displays an error message that applicant already exists. <br>
     Use case resumes at step 1.
 
 --------------------------------------------------------------------------------------------------------------------
 
-**Use case: UC02 - Delete an applicant**
+**Use case: UC03 - List all applicants**
+
+**MSS**
+
+1. User requests to list all applicants.
+2. TrackAScholar shows the list of all applicants. <br>
+   Use case ends.
+
+**Extensions**
+
+* 1a. Input format is wrong.
+
+    * 1a1. TrackAScholar displays an error message prompting correct input format. <br>
+      Use case resumes at step 1.
+  
+* 1b. Input format has extra parameters.
+    * 1b1. TrackAScholar ignores the extra parameters. <br>
+      Use case resumes at step 2.
+
+* 1c. TrackAScholar has no applicants stored.
+
+    * 1c1. TrackAScholar shows an empty applicant list.
+
+--------------------------------------------------------------------------------------------------------------------
+
+**Use case: UC04 - Edit personal details of an applicant**
 
 **Guarantees:**
-* Once an applicant is deleted,the applicant will be removed from the database and the user will no longer
-be able to view the applicant from the list.
+
+* An applicant's name cannot be updated to an applicant name that already exists.
 
 **MSS**
 
-1. User request to delete an applicant in TrackAScholar.
-2. TrackAScholar removes applicant with matching name.
+1. User requests to change the details of an applicant (such as email or contact number etc.).
```

Comment:
use ***phone number*** instead of contact number

Reply:
Removed the brackets as too much detail in implementation.